python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -66,6 +73,21 @@ class HistPanel(Config):
         "is plotted, the percentile limit is the maximum value across all input data.",
         default=98.0,
     )
+    expectedValue = Field[float](
+        doc="Value at which to add a black solid vertical line.  Ignored if set to None.",
```

Comment:
Double space after period.

Reply:
:point_right: https://developer.lsst.io/python/style.html#sentences-in-comments-should-not-be-separated-by-double-spaces :wink: :smile:

Reply:
I stand corrected!!  Perhaps the template needs updating to meet our standards 😆 (and I did try to search for this in the dev guide...my key word "period" resulted in me missing this!)

Reply:
All of the offending double spaces should now be removed!